taskcluster/fxci_config_taskgraph/transforms/integration_test.py

```diff
@@ -144,7 +205,51 @@ def rewrite_private_fetches(taskdesc: dict[str, Any]) -> None:
             payload["env"]["MOZ_FETCHES"] = {"task-reference": json.dumps(fetches)}
 
 
-def make_integration_test_description(task_def: dict[str, Any], name_prefix: str):
+def rewrite_mirrored_dependencies(
```

Comment:
A docstring would be nice. From a glance, I don't know what a "mirrored" dependency is.